packages/block-editor/src/components/rich-text/format-toolbar-container.js

```diff
@@ -19,9 +24,12 @@ const FormatToolbarContainer = ( { inline, anchorRef } ) => {
 				focusOnMount={ false }
 				anchorRef={ anchorRef }
 				className="block-editor-rich-text__inline-format-toolbar"
-				__unstableSlotName="block-toolbar"
+				// Render inline
+				__unstableSlotName={ null }
```

Comment:
I'm passing `null` to the slot name so it renders inline in the React tree, and not on a separate slot as it won't find any slot: https://github.com/WordPress/gutenberg/blob/5c24a7d6e5882a1f2b9e0dff787fecfccce01335/packages/components/src/popover/index.js#L627-L635

I don't know if there's a better way to achieve this, but I think we could have an `inline` prop or something on Popover.